packages/modules/atlas-core/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this widget will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- We fixed that the navsidebar sub-menu starts at the same height as the sidemenu itself
+
+### Changed
+- Submenu on the navsidebar desktop now opens on the same height as the main menu
```

Comment:
The **Changed** section is now duplicated, please rewrite the commit which introduces this.